docs/cms.rst

```diff
@@ -112,6 +112,68 @@ This approach will not be a problem if we stick to image filter-specs from the
 'approved' list. Note that extending the list of filter-specs is possible, if
 we need to.
 
+L10N and Translation Management
+-------------------------------
+
+.. important::
+
+    Localization via Wagtail is something we are ramping up on, so please
+    do not assume the following notes are final, or that the workflows are
+    currently all rock-solid. We're learning as we go.
+
+Page-tree concept
+=================
+
+Our Wagtail setup uses the official `wagtail-localize`_ package to manage
+localization of pages.
+
+This package supports page-level localization rather than field-level localization, which means that each locale has its own distinct tree of pages, rather than each page having a stack of duplicate fields, one per destination language.
+
+These language-specific trees can be "synchronised" with the default ``en-US`` page tree, so would have the same page structure, field by field) — or they can not be synchronised, so can have their own extra pages, or some specific pages in the tree can be made not "synchronised", while others are.
+
+Basically, there is plenty of flexibility. The flipside of that flexibility is we may also create an edge-case situation that ``wagtail-localize`` won't work with, but we'll have to see and deal with it.
+
+.. note::
+
+    It's worth investing 15 mins in watching the `Wagtail Localize original demo`_ to get a good feel of how it can work.
```

Comment:
Just noticing the above text looks like it's wrapped whereas this new text isn't.